PyRoute/Star.py

```diff
@@ -155,13 +155,25 @@ def __getstate__(self):
             del state['ownedBy']
         return state
 
+    def __deepcopy__(self, memodict={}):
```

Comment:
Star needed an explicit __deepcopy__ implementation because __getstate__ got in the way of star filtering for stats generation (iirc, high tech worlds in a (sub)sector).